sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -70,11 +72,13 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
 
   try {
     sparkSession.sparkContext.conf.get(STREAMING_QUERY_LISTENERS).foreach { classNames =>
-      Utils.loadExtensions(classOf[StreamingQueryListener], classNames,
-        sparkSession.sparkContext.conf).foreach(listener => {
-        addListener(listener)
-        logInfo(s"Registered listener ${listener.getClass.getName}")
-      })
+      SQLConf.withExistingConf(sqlConf) {
+        Utils.loadExtensions(classOf[StreamingQueryListener], classNames,
+          sparkSession.sparkContext.conf).foreach(listener => {
```

Comment:
Can we remove outer `(...)`? e.g.) `.foreach { ...`

Reply:
It seems to be a legacy issue, modified